tests/cxx/unit_tests/parallelzone/runtime/detail_/runtime_view_pimpl.cpp

```diff
@@ -73,4 +73,19 @@ TEST_CASE("RuntimeViewPIMPL") {
             REQUIRE_FALSE(pimpl == other);
         }
     }
+
+    SECTION("stack_callback") {
+        // Simulate initialization
+        bool is_running = true;
+
```

Comment:
I'd recommend testing for stack behavior, i.e., push back two lambdas and have the first ensure that the second one was called first.